app/realtime/card_realtime.rb

```diff
@@ -23,7 +23,7 @@ def self.card_archived(card)
     )
   end
 
-  def self.card_previewed(card, preview_url)
+  def self.card_previewed(card, originated_by, preview_url)
```

Comment:
Unused method argument - `originated_by`. If it's necessary, use `_` or `_originated_by` as an argument name to indicate that it won't be used.